### PR TITLE
Enable the ability to turn on firewall logs and add labels to resources

### PR DESCRIPTION
**Pull request by upodroid**
Hi

I have a requirement to enable firewall logging on rules that is created in our network. Please accept this change and release a new patch version once it has been merged.

Thank you

### CODE REVIEW

**Comment by upodroid** [comment]
I will also need to set labels on all created resources. I'll push a commit that can do that later today.

**Comment by gkowalski-google** [comment]
Hi @upodroid, your changes look good to me. Let me fix the build triggers so the tests run and I can approve afterwards.

**Comment by upodroid** [comment]
I'm ready to merge this now, I have:
- added labels to gcs, cloudsql and instances
- versions.tf now supports terraform 0.13 syntax but requires 0.12+. I'm currently using the beta and it is great.

**Comment by upodroid** [comment]
Another thing i just noticed, forseti service account is missing metric writer in the project. I'm thinking both the client and the server should push metrics to Stackdriver.

image

**Comment by upodroid** [comment]
image

Much better now :)

File: modules/client_gcs/variables.tf [hunk 1]
```diff
@@ -49,3 +49,9 @@ variable "services" {
   type        = list(string)
   default     = []
 }
+
+variable "gcs_labels" {
+  description = "GCS bucket labels"
+  type        = map(string)
+  default     = {}
+}
```
Comment (gkowalski-google) [inline modules/client_gcs/variables.tf:57]: (nit) Add newline.

File: modules/cloudsql/variables.tf [hunk 2]
```diff
@@ -99,3 +99,9 @@ variable "cloudsql_availability_type" {
   description = "Whether instance should be set up for high availability (REGIONAL) or single zone (ZONAL)."
   default     = null
 }
+
+variable "cloudsql_labels" {
+  description = "CloudSQL instance labels"
+  type        = map(string)
+  default     = {}
+}
```
Comment (gkowalski-google) [inline modules/cloudsql/variables.tf:107]: (nit) Add newline.

File: modules/real_time_enforcer_organization_sink/outputs.tf [hunk 3]
```diff
@@ -24,3 +24,7 @@ output "topic" {
   value       = google_pubsub_topic.main.name
 }
 
+output "writer_identity" {
+  description = "The identity of the sink writer"
+  value       = google_logging_organization_sink.main.writer_identity
+}
```
Comment (gkowalski-google) [inline modules/real_time_enforcer_organization_sink/outputs.tf:30]: (nit) Add newline.

**Comment by gkowalski-google** [comment]
@upodroid Here's the error from the tests. It doesn't seem to be related to your changes, perhaps it is a race condition and we just need to update the `forseti-private-zone` to depend on the [google_project_service](https://github.com/forseti-security/terraform-google-forseti/blob/master/examples/private_connection/main.tf#L32) resource?

```
Error: Error creating ManagedZone: googleapi: Error 403: Cloud DNS API has not been used in project {{ PROJECT_NUMBER }} before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/dns.googleapis.com/overview?project={{ PROJECT_NUMBER }} then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry., accessNotConfigured
       
         on ../../../examples/private_connection/main.tf line 83, in resource "google_dns_managed_zone" "forseti-private-zone":
         83: resource "google_dns_managed_zone" "forseti-private-zone" {
```

**Comment by upodroid** [comment]
Looks it is ready to be merged now.

**Pull request merged**